x-pack/plugins/ml/public/file_datavisualizer/components/file_datavisualizer_view/file_error_callouts.js

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+
+import {
+  EuiCallOut,
+} from '@elastic/eui';
+
+export function FileTooLarge({ fileSize, maxFileSize }) {
+  return (
+    <EuiCallOut
+      title="File size is too large"
+      color="danger"
+      iconType="cross"
+    >
+      <p>
+        File size uploaded is {fileSize}, the max file size for uploading to Kibana is {maxFileSize}
```

Comment:
Might read better as e.g. The size of the file you selected for upload is {fileSize} which exceeds the maximum of {maxFileSize} permitted by Kibana